src/Corax/Pipeline/Parsing/VectorTokenizers.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace Corax.Pipeline.Parsing
+{
+    internal static class VectorTokenizers
+    {
+        public static int TokenizeWhitespaceAsciiSse(ReadOnlySpan<byte> buffer, ref Span<Token> tokens)
```

Comment:
Can you add this to the testbase?
```csharp
[RavenMultiplatformTheory(RavenTestCategory.Corax, RavenIntrinsics.Sse)]
        [InlineData("🚁🚁🚁🚁🚁 🚁🚁🚁🚁🚁", 2, new string[]{"🚁🚁🚁🚁🚁", "🚁🚁🚁🚁🚁"})] // 4bytes, adjusted to fit in Vec128
        [InlineData("‰‰‰‰‰‰ ‰‰‰‰‰‰", 2, new string[]{"‰‰‰‰‰‰", "‰‰‰‰‰‰"})] // 3 bytes
        [InlineData("ΣΣΣΣΣΣΣΣ ΣΣΣΣΣΣΣΣ", 2, new string[]{"ΣΣΣΣΣΣΣΣ", "ΣΣΣΣΣΣΣΣ"})] // 2 bytes
        public void WhitespaceTokenizerUtf8TestSse(string stringToCheck, int expectedTokens, string[] expectedWords)
        {
            Span <Token> tokens = stackalloc Token[4];
            Span<byte> buffer = Encodings.Utf8.GetBytes(stringToCheck);
            VectorTokenizers.TokenizeWhitespaceAsciiSse(buffer, ref tokens);
            Assert.Equal(expectedTokens, tokens.Length);
            for (int i = 0; i < expectedTokens; i++)
            {
                var token = tokens[i];
                var innerWord = buffer.Slice(token.Offset, (int)token.Length);
                var innerAsStr = Encoding.UTF8.GetString(innerWord);
                Assert.Equal(expectedWords[i], innerAsStr);
            }
        }
```

Reply:
This is a Whitespace Tokenizer, we only care about the size of the whitespaces. Beyond the ASCII one, the only ones we care about are of size 2.